webapp-django/crashstats/crashstats/static/crashstats/css/base/layout.less

```diff
@@ -173,6 +173,12 @@ body {
     }
 }
 
+.protected-info {
+    margin: 1em 1em 1em;
+    display: flex;
+    align-items: center;
+}
```

Comment:
This is a new element that's added to the header on relevant pages to show the protected data status of the page and why/why not a user is seeing protected data.